test/specs/scenarios/proofOfAddress.js

```diff
@@ -141,6 +141,7 @@ export const proofOfAddressScenarios = async(lang='en') => {
       verificationComplete.verifyUIElements(copy)
     })
 
+    // FIXME
```

Comment:
Why is this failing?

Reply:
Oops. My bad. Forgot to remove this `FIXME` after I updated the test.

For context, it was originally failing (as expected) because of the changes to the UI with this update.